libc/src/unistd/pread.h

```diff
@@ -9,8 +9,8 @@
 #ifndef LLVM_LIBC_SRC_UNISTD_PREAD_H
 #define LLVM_LIBC_SRC_UNISTD_PREAD_H
 
+#include "hdr/unistd_macros.h"
```

Comment:
could you apply this to the other unistd functions as well?

Reply:
Done.